### ISSUE TITLE
Reduce warnings...

### ISSUE THREAD

**Issue by frederikhors** [pateketrueke/yrv#40]
In order to reduce `<PageX> was created with unknown prop 'router'` warning can we use something like `svelte-spa-router`?

https://github.com/ItalyPaleAle/svelte-spa-router/blob/master/Router.svelte#L207-L211.

What do you think about?

**Comment by frederikhors** [comment]
Thinking about it, we can't.

Because props are different.

**Comment by frederikhors** [comment]
Right?

**Comment by pateketrueke** [comment]
If we just remove such prop? I mean, we _can_ live using the `$router` store only. 🤔 

Comparing the needed boilerplate:

```js
import { router } from 'yrv';
console.log($router.path);
```

vs

```js
export let router = null;
console.log(router.path);
```

I'm not aware of the implications, but the change is fairly simple.

**Comment by frederikhors** [comment]
Oh dear. This is amazing!

**Comment by frederikhors** [comment]
So we are just using Svelte stores, right?

**Comment by pateketrueke** [comment]
> So we are just using Svelte stores, right?

Yes! Actually there is a subtle difference, the `let router` has three additional props, but they are completely optional so I think it's fine to omit them completely.

If we change that, we need to remote the `let:router` support too from slots to keep things consistent I guess.
